tests/tt_eager/python_api_testing/sweep_tests/pytests/tt_dnn/wormhole/test_bert_binary.py

```diff
@@ -20,8 +20,10 @@
 def run_binary_bert_tests(tt_op, pt_op, input_shapes, dtype, dlayout, in_mem_config, out_mem_config, data_seed, device):
     torch.manual_seed(data_seed)
 
-    x = torch.Tensor(size=input_shapes[0]).uniform_(-100, 100)
-    y = torch.Tensor(size=input_shapes[1]).uniform_(-100, 100)
+    x = gen_rand_complex(size=input_shape, low=-100, high=100).to(torch.bfloat16)
```

Comment:
Fails with gen_rand_complex `NameError: name 'gen_rand_complex' is not defined`. Probably `gen_rand_complex ` should not be used in bert tests.